index.html

```diff
@@ -289,6 +289,33 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+<section class="contact-me">
+  <div>
+  <p class="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+  </div>
+  <div>
+  <form action="https://formspree.io" method="post">
+  <input type="text" id="name" name="name" placeholder="Full Name" maxlength="90" required>
+  <input type="text" id="email" name="email" placeholder="Email Addres" required>
+  <textarea name="textarea" placeholder="Enter text here" required></textarea>
+  <div><button id="footer-button" type="submit">Get in touch</button></div>
+  </form>
```

Comment:
- [ ] Kindly check for the maxlegth value for the input tag corresponding to the name, it should be set as 30.

- [ ] Kindly add  the maxlegth value and attribute on the input tag corresponding to the text area it should be set as 500.